cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/NamespaceHttpHandler.java

```diff
@@ -16,13 +16,12 @@
 
 package co.cask.cdap.gateway.handlers;
 
-import co.cask.cdap.app.store.Store;
-import co.cask.cdap.app.store.StoreFactory;
 import co.cask.cdap.common.conf.Constants;
-import co.cask.cdap.config.DashboardStore;
-import co.cask.cdap.config.PreferencesStore;
+import co.cask.cdap.common.exception.AlreadyExistsException;
```

Comment:
These exceptions are in ``cdap-common``. I'll create a separate PR to move them to ``co.cask.cdap.common`` package. Is that ok?